crates/weaver_version/src/metrics_changes.rs

```diff
@@ -1,11 +1,19 @@
 // SPDX-License-Identifier: Apache-2.0
 
-//! Metrics change definitions.
+//! Metrics version.
```

Comment:
nit: I think "change definitions" was better here...

Reply:
I don’t recall any strong reasons for making this change. I’ll revert to the previous comment.

Reply:
Fixed